scenes/MosaikHeer.scn

```diff
@@ -547,14 +547,14 @@
 /ch/17/preamp -18.0 OFF ON 24 114
 /ch/17/gate OFF GATE -80.0 60.0 1  502  983 0
 /ch/17/gate/filter OFF 3.0 990.9
-/ch/17/dyn ON COMP PEAK LOG -20.5 3.0 4 3.50 22 0.02  37 POST 0 100 OFF
+/ch/17/dyn OFF COMP PEAK LOG -20.5 3.0 4 3.50 22 0.02  37 POST 0 100 OFF
 /ch/17/dyn/filter ON 2.0 1k97
-/ch/17/insert OFF POST OFF
+/ch/17/insert ON POST FX5L
```

Comment:
I am not a big fan of this compressor. However, I would try to play around with it again to get more familiar with it. Thus, can we set the standard to OFF? Everyone can enable it if they want. Can you make sure, you add this to the PR description?

Reply:
Okay, agreed. I'll turn it off by default and the channel compressor on again.